syntax = "proto3";

package hello_service_proto;

message HelloRequest {
    string name = 1;
}

message HelloReply {
    string message = 1;
}

service HelloService {
    rpc GetReply (HelloRequest) returns (HelloReply) {}
}
